src/main/java/rs/teslaris/core/service/impl/person/PersonContributionServiceImpl.java

```diff
@@ -63,11 +63,12 @@ public void setPersonDocumentContributionsForDocument(Document document,
     }
 
     @Override
-    public void setPersonJournalContributionsForJournal(Journal journal, JournalDTO journalDTO) {
-        if (journal.getContributions() != null) {
-            journal.getContributions().clear();
+    public void setPersonJournalContributionsForJournal(PublicationSeries publicationSeries,
```

Comment:
I believe we need to rename this method (and to slightly change) or to create the another one for BookSeries (the another subclass of PublicationSeries)

Reply:
Publication Series Contributions are part of publication series class for now. If it stays like that then I think just renaming the method to `setPersonPublicationSeriesContributionsForJournal` and making it work with `PersonPublicationSeriesContribution` class should be sufficient.